model/Image.php

```diff
@@ -1051,3 +1051,14 @@ public function write($showDebug = false, $forceInsert = false, $forceWrite = fa
 		throw new Exception("{$this->ClassName} can not be written back to the database.");
 	}
 }
+
```

Comment:
I don't think this is the right place to add new global methods. These should be in `Convert` class as static methods instead. Perhaps consider including the json_en/decoding in the method as well as a generic 'encode array as url safe text'